lib/phlex/view.rb

```diff
@@ -36,6 +50,8 @@ def call(buffer = +"", view_context: nil, parent: nil, &block)
 			@_parent = parent
 			@output_buffer = self
 
+			block ||= -> {}
+			block.instance_variable_set(:@rendering, true)
```

Comment:
Ensure we always have a block, even if it doesn't do anything and set the instance variable `@rendering` on it.